src/module/entities/helper.js

```diff
@@ -578,7 +581,7 @@ module.exports = class UserProjectsHelper {
 	 * @param {Array} [entityIds = false] - Array of entity ids.
 	 */
 
-	static search(searchText, pageSize, pageNo, entityIds = false) {
+	static search(searchText, pageSize, pageNo, entityIds = false, language) {
```

Comment:
@MallanagoudaB update function signature. If language is not passed wee should return default data